README.md

```diff
@@ -1,33 +1,94 @@
-* <a href="https://virtocommerce.com/docs/vc2devguide/working-with-storefront/pages-and-blogs" target="_blank">Pages and Blogs</a>
+1. Theme;
+1. Pages;
+1. Link lists;
+1. Blogs.
+
+![Content Module](docs/media/screen-content-module.png)
+
+## Key features
+
+1. Themes Management on UI;
+1. Pages and blogs management on UI;
+1. Link lists management;
+1. Manage documents in markdown format.
+
+The VC Content module can be accessed both ways: directly and via Stores module.
+
+To access the VC Content directly you should navigate to More->Content Module
+
+![Direct Access](docs/media/screen-access-content-directly.png)
+
+To access the VC Content Module via Stores module, you should navigate to More->Stores->Store name-> Content module widget
+
+![Access from Store Module](docs/media/screen-access-from-store-module.png)
+
+## Feature Description
+
+### Virto Commerce Theme
+
+Each theme template comes with its own settings. These settings allow you to customize your ecommerce website's design without requiring HTML or CSS knowledge.
+
+Change logo, fonts, colors and more. Make your ecommerce website have the look you like.
+
+You can also customize homepage and product layouts, add banners, ads, slideshows and more.
+
+Your ecommerce website can be displayed in any language you want. You can allow customers to pick a language they want or auto detect based on location. Virto Commerce supports local currencies, tax rates, prices etc.
+
+You get full control over all the ecommerce templates and can translate them to any language. If page doesn’t have any translation available then default one can be used instead.
+
+Depending on the theme used, the following translations are available: Danish, Dutch, Spanish, French, German, Greek, Italian, Japanese, Portuguese, Chinese, Swedish and over 50 other languages.
+
+Virto Commerce gives you 100% control over the design and HTML/CSS used by your ecommerce website and online shopping cart.
+
+[Theme Management](/docs/theme-management.md)
+
+### Pages and Blogs
+
+Virto Commerce implements a variation of so called NO-CMS approach (similar to Jekyll) for pages and blogs (and themes to some extent). That means there is no hard dependency on where CMS content like pages, articles and templates are stored. They can be stored in database, github or local file system. This can be configured using CMS config. The content is downloaded from the remote location at runtime and is saved in the local website folder structure under App_Data folder from where local runtime generator picks it up and renders as html.
+
+Pages and blog articles are created using templating engine that supports both markdown and liquid. During runtime those templates are converted to html and persisted in memory until files are changed at which point content is regenerated.
+
+[Pages and Blogs Management](/docs/pages-blogs-management.md)
```

Comment:
How to edit Pages and blogs in Store described in the [Pages and Blogs Management](/docs/pages-blogs-management.md) article